handsontable.d.ts

```diff
@@ -705,26 +705,32 @@ declare namespace Handsontable {
       setSampleCount(sampleCount: number): void;
     }
 
-    class Base {
+    class BasePlugin {
       pluginName: string;
       pluginsInitializedCallback: any[];
       isPluginsReady: boolean;
       enabled: boolean;
       initialized: boolean;
+      hot: any;
```

Comment:
Why `any`? Isn't it Handsontable instance?

Reply:
You right, I changed this.